releasenotes/notes/rds-credentials-1fa9817a21a2d80a.yaml

```diff
@@ -0,0 +1,3 @@
+---
+features:
+  - Add AWS credential source configurations to RDS functions
```

Comment:
`Enable RDS functions to accept AWS credentials using a Secret or ServiceAccount.`

WDYT?

Reply:
Yes, it's better.